packages/core/phase1/triggers/TRPS0013.test.ts

```diff
@@ -87,23 +91,23 @@ describe("TRPS0013", () => {
   })
 
   it.each([
-    { addedByCourt: false, isRecordable: false, hasCompletedDisarr: false, disarrCompatibleResultClass: false },
-    { addedByCourt: true, isRecordable: false, hasCompletedDisarr: false, disarrCompatibleResultClass: false },
-    { addedByCourt: false, isRecordable: true, hasCompletedDisarr: false, disarrCompatibleResultClass: false },
-    { addedByCourt: false, isRecordable: false, hasCompletedDisarr: true, disarrCompatibleResultClass: false },
-    { addedByCourt: false, isRecordable: false, hasCompletedDisarr: false, disarrCompatibleResultClass: true },
-    { addedByCourt: true, isRecordable: true, hasCompletedDisarr: false, disarrCompatibleResultClass: false },
-    { addedByCourt: true, isRecordable: true, hasCompletedDisarr: true, disarrCompatibleResultClass: false },
-    { addedByCourt: false, isRecordable: true, hasCompletedDisarr: true, disarrCompatibleResultClass: true },
-    { addedByCourt: false, isRecordable: false, hasCompletedDisarr: true, disarrCompatibleResultClass: true },
-    { addedByCourt: true, isRecordable: false, hasCompletedDisarr: false, disarrCompatibleResultClass: true },
-    { addedByCourt: true, isRecordable: false, hasCompletedDisarr: true, disarrCompatibleResultClass: false },
-    { addedByCourt: false, isRecordable: true, hasCompletedDisarr: false, disarrCompatibleResultClass: true },
-    { addedByCourt: true, isRecordable: true, hasCompletedDisarr: false, disarrCompatibleResultClass: true },
-    { addedByCourt: true, isRecordable: false, hasCompletedDisarr: true, disarrCompatibleResultClass: true }
+    { isRecordable: false, hasCompletedDisarr: false },
+    { isRecordable: false, hasCompletedDisarr: false },
+    { isRecordable: true, hasCompletedDisarr: false },
+    { isRecordable: false, hasCompletedDisarr: true },
+    { isRecordable: false, hasCompletedDisarr: false },
+    { isRecordable: true, hasCompletedDisarr: false },
+    { isRecordable: true, hasCompletedDisarr: true },
+    { isRecordable: true, hasCompletedDisarr: true },
+    { isRecordable: false, hasCompletedDisarr: true },
+    { isRecordable: false, hasCompletedDisarr: false },
+    { isRecordable: false, hasCompletedDisarr: true },
+    { isRecordable: true, hasCompletedDisarr: false },
+    { isRecordable: true, hasCompletedDisarr: false },
+    { isRecordable: false, hasCompletedDisarr: true }
```

Comment:
This could be reduced to only 4 scenarios now that we've removed the redundant parameters.